nupic/research/frameworks/vernon/handlers.py

```diff
@@ -1133,6 +1135,7 @@ def setup_experiment(self, config):
         self.val_batch_size = config.get("val_batch_size", 15)
         self.slow_batch_size = config.get("slow_batch_size", 64)
         self.replay_batch_size = config.get("replay_batch_size", 64)
+        self.num_fast_steps = config.get("num_fast_steps", 1)
```

Comment:
The name is a bit misleading, since it will only take as much as steps as the number of (samples in class / batch_size). To effectively take the number of steps requested by the user it would have to do as many loops through the loader as required, but that is not what is happening in the train function below. 

Maybe we can add a message stating the effective number of fast steps in case the number given by the user is greater than (samples in class / batch_size)

Reply:
Good point, I'll add a warning.